README.md

```diff
@@ -165,6 +165,12 @@ augroup CoqtailHighlights
 augroup END
 ```
 
+If you feel distracted by the error highlighting while editing a failed
+sentence, you can use the sequence `<leader>ci` (`:CoqToLine`) while the cursor
```

Comment:
```suggestion
sentence, you can use the sequence `<leader>cl` (`:CoqToLine`) while the cursor
```

The default mapping for `:CoqToLine` is `<leader>cl`.

Reply:
I thought I took care of that while rebasing… (I have custom mappings as an additional commit.)